views/lang/latin/verb/latin-verb-irregular.js

```diff
@@ -0,0 +1,80 @@
+import { Constants, LanguageModelFactory, Feature } from 'alpheios-data-models'
+import LatinView from '@views/lang/latin/latin-view.js'
+
+import GroupFeatureType from '@views/lib/group-feature-type'
+import Form from '@lib/form.js'
+import Table from '@views/lib/table'
+
+export default class LatinVerbIrregularView extends LatinView {
+  constructor (inflectionData, locale) {
+    super(inflectionData, locale)
+
+    this.id = 'verbConjugationIrregular'
+    this.name = 'verb-irregular'
+    this.title = 'Verb Conjugation (Irregular)'
+
+    const inflectionsWords = inflectionData.homonym.inflections.map(item => item[Feature.types.word].value)
+    const lemma = this.dataset.verbsIrregularLemmas.filter(item => inflectionsWords.indexOf(item.word) > -1)[0]
+
+    this.additionalTitle = lemma.word + ' - ' + lemma.principalParts
```

Comment:
@irina060981 please just join the lemma.word to the principal parts with a comma rather than a hyphen and then this is great to merge. Thank you!